src/main/java/com/shopify/model/ShopifyGetCustomersRequest.java

```diff
@@ -0,0 +1,135 @@
+package com.shopify.model;
+
+import org.joda.time.DateTime;
+
+import java.util.List;
+
+public class ShopifyGetCustomersRequest {
+    private int page;
+    private int limit;
+    private List<String> ids;
+    private String sinceId;
+    private DateTime createdAtMin;
+    private DateTime createdAtMax;
+    public static interface OptionalsStep {
```

Comment:
Awesome job here! Thanks!

Reply:
Thanks! Anything else you want me to do before merging? We're interested in using this SDK for a project at the company I work for, so I'm happy to do anything I need to to get these changes merged and released.

Also, I'm noticing that after running `mvn clean install` and moving the jar to the project I'm working on, I'm getting a `java.lang.NoClassDefFoundError: org/glassfish/jersey/jackson/JacksonFeature`. Any insight here? The only thing I could gather online is that bumping the version might help, but it hasn't.